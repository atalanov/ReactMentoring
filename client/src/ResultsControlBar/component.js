// @flow
import React from 'react';
import ResultsCount from '../ResultsCount';
import ResultsSorting from '../ResultsSorting';

export default props => (
  <div className="row">
    <div className="col-8">
      <ResultsCount />
    </div>
    <div className="col-4 text-right">
      <ResultsSorting />
    </div>
  </div>
);
