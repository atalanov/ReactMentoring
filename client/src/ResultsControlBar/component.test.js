import React from 'react';
import { shallow } from 'enzyme';
import ResultsControlBar from './component';

test('Component ResultsControlBar renders correctly', () => {
    const component = shallow(<ResultsControlBar />);
    expect(component).toMatchSnapshot();
});
