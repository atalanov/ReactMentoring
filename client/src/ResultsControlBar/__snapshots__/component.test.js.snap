// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ResultsControlBar renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="col-8"
        >
          <Connect(ResultsCount) />
        </div>,
        <div
          className="col-4 text-right"
        >
          <Connect(ResultSorting) />
        </div>,
      ],
      "className": "row",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Connect(ResultsCount) />,
          "className": "col-8",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Connect(ResultSorting) />,
          "className": "col-4 text-right",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="col-8"
          >
            <Connect(ResultsCount) />
          </div>,
          <div
            className="col-4 text-right"
          >
            <Connect(ResultSorting) />
          </div>,
        ],
        "className": "row",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Connect(ResultsCount) />,
            "className": "col-8",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Connect(ResultSorting) />,
            "className": "col-4 text-right",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
